livre/inc/patterns/footer-nav-left.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Footer with left navigation block pattern
+ */
+return array(
+	'title'      => __( 'Footer with left navigation', 'livre' ),
```

Comment:
I believe this one should be labeled "Footer with right navigation".

Reply:
😄 Whoops, good catch. Should be sorted out in 4788105.